frontend/src/metabase/components/Header.jsx

```diff
@@ -143,14 +143,9 @@ class Header extends Component {
       );
     }
 
-    let attribution;
-    if (this.props.item && this.props.item.creator) {
-      attribution = (
-        <div className="Header-attribution">
```

Comment:
The class name `Header-attribution` is added in https://github.com/metabase/metabase/pull/2136. Alongside, CSS that hid it.
https://github.com/metabase/metabase/pull/2136/files#diff-70dba2e3e8a73fb80938e1c5e8414d87d5bac4bff80b00708420999b726b9f95R11-R15

So, if we don't render it (`display: none;`) we might as well remove it.